ttnn/cpp/ttnn/operations/data_movement/pad/device/pad_program_factory.cpp

```diff
@@ -1290,9 +1290,6 @@ inline std::vector<std::pair<std::vector<uint32_t>, std::vector<uint32_t>>> get_
                 reader_kernel_args.push_back((std::uint32_t) xy_pair.first); // noc x
                 reader_kernel_args.push_back((std::uint32_t) xy_pair.second); // noc y
             }
-
```

Comment:
are changes in this file accidental?

Reply:
No, these logs are not useful and being in a loop, they output a very large amount of logs, which drown everything else.